src/mongo_logger.ts

```diff
@@ -281,7 +285,8 @@ export interface Log extends Record<string, any> {
 
 /** @internal */
 export interface MongoDBLogWritable {
-  write(log: Log): void;
+  write(log: Log): PromiseLike<unknown> | any;
+  streamName?: string;
```

Comment:
This type is intended to go public and be the alternative to specifying stderr/out, I do not think we want users to be required to specify this property. Can the stream destination be tracked on the logger class instead to keep this out of the public API?

Reply:
Yes, it can be tracked in the logger class;  `streamName` is now removed from the public API.